client/my-sites/customer-home/cards/notices/celebrate-notice-v2/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import { requestHomeLayout } from 'state/home/actions';
+import { savePreference } from 'state/preferences/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import fireworksIllustration from 'assets/images/illustrations/fireworks.svg';
+
+const CelebrateNotice = ( {
+	actionText,
+	description,
+	noticeId,
+	onSkip,
+	showSkip = false,
+	skipText,
+	siteId,
+	title,
+} ) => {
+	const [ isVisible, setIsVisible ] = useState( true );
+	const dispatch = useDispatch();
+
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	const dismissalPreferenceKey = `dismissible-card-home-notice-${ noticeId }-${ siteId }`;
+
+	const showNextTask = () => {
+		setIsVisible( false );
```

Comment:
Not sure what I think about this. It works great, but gives the users a jumping UI, and a moment while it loads that there's no indicator of what's going on. I'd almost prefer we just set the preference and do a page reload; I think the user would better understand what's happening. Another option would be showing the placeholder again after `setIsVisible`; users would recognize that from the initial page load and understand there's more coming.

Reply:
> we just set the preference and do a page reload;

No need for a hard page reload, onclick, it'd be a POST or another GET fetch to /layout.

Reply:
> No need for a hard page reload, onclick, it'd be a POST or another GET fetch to /layout.

That's the existing behaviour we have here now. Maybe I'm overestimating the UX confusion; the load happens quite quickly.

Reply:
Sorry, commented without peeking at code. I suppose if there was a slow internet connection we could drop `setIsVisible( false );` or add a pulsing placeholder, but this is likely sometime we can follow up on if it's an issue.

Reply:
Hmm on second thought, @danhauk what do you think? Maybe add a pulsing bar or some other transition? It would be a little unexpected if it takes too long.

![May-04-2020 15-29-46](https://user-images.githubusercontent.com/1270189/81019793-3ce39e00-8e1c-11ea-8d28-a0a106e51fee.gif)

Reply:
@gwwar I agree about the jumping UI and a transition would be nice to have there. Could we try the page load pulsing bar to start and see how that looks?

![my home loading](https://user-images.githubusercontent.com/448298/81335092-0bb1da80-9075-11ea-9cf0-4b0027d1d0a8.gif)

Reply:
Sure @danhauk, should we keep height there a bit bigger? I'm mostly worried about folks trying to interact with items below the primary area and things popping in.

Reply:
We could also overlay the celebration notice with a spinner; that way any jumping will be limited to the height difference between the celebration notice and the new task card.

Reply:
Feel free to try that Kirk and see how it feels, we might end up needing to prototype and see which approach works out better in practice

Reply:
Pushed a commit exploring the approach suggested by @kwight:

![May-11-2020 16-21-55](https://user-images.githubusercontent.com/1233880/81572630-b8f15f00-93a3-11ea-9a5d-412690ec5e95.gif)

Reply:
> Pushed a commit exploring the approach suggested by @kwight

@mmtr Hey, that doesn't look bad at all – what's going on is understandable from a user's point of view, I think. A good starting point, at least 🎉

Reply:
@mmtr This looks pretty good as a starting point. It would be great for a future iteration to use a skeleton loader if we know what's coming next (i.e. a skeleton of the site setup list after the site creation banner, a skeleton of a task after the site launch, etc.)

<img width="1061" alt="image" src="https://user-images.githubusercontent.com/448298/81843597-cdf0fe00-951b-11ea-87dd-a254af74dca7.png">